src/app.py

```diff
@@ -9,13 +13,26 @@ def say_hello_world():
     return {'result': "Flask says Hello World"}
 
 @app.route('/surgeo', methods=["GET"])
-def surgeo():
+def run_surgeo():
     # TO-DO: @KAYLA
     # Build out a public facing API that responds to a get request with the surname and zip code as query arguments. 
     # The API should return the probabilities of each race prediction as a JSON object.
     # Example of return object: {'AAPI': .56, 'Hispanic': .32, ..., 'White': .10}
 
-    pass
+    surname = request.args.get('surname')
+    zipcode = request.args.get('zipcode')
+    surname_series = pd.Series([surname])
+    zip_series = pd.Series([zipcode]) 
+
+    sg = surgeo.SurgeoModel()
+    sg_results = sg.get_probabilities(surname_series, zip_series)
+
+    # print for debugging
```

Comment:
Could you please remove these debug statements? In general, when merging to 'dev', all temporary debugging code should be removed, to keep the main branch clean. 

Sidenote: comments are a little more nuanced. Feel free to keep in comments that you think are useful for informing the other engineers and explaining the code. But please take out all comments that are mostly for yourself (such as reminders).